ibm/resource_ibm_pi_instance.go

```diff
@@ -114,13 +114,19 @@ func resourceIBMPIInstance() *schema.Resource {
 				Description: "Base64 encoded data to be passed in for invoking a cloud init script",
 			},
 
-			"pi_storage_type": {
+			helpers.PIInstanceStorageType: {
 				Type:        schema.TypeString,
 				Optional:    true,
 				Computed:    true,
 				Description: "Storage type for server deployment",
 			},
 
+			helpers.PIInstanceStorageConnection: {
+				Type:        schema.TypeString,
+				Optional:    true,
```

Comment:
Can you add validate func if the supported value is only `vSCSI`

Reply:
done